tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+- name: Install role requirements
+  import_tasks: requirements.yml
+  when: install_home_assistant_supervised_ansible_requirements|bool
+
+- name: Install OS package prereqs
+  apt:
+    name:
+      - systemd
+      - network-manager
+      - apparmor
+      - jq
+      - curl
+      - dbus
+    state: latest
+    update_cache: yes 
+    
+- name: Disable Modem Manager to prevent issues when using serial devices
+  service:
+    name: ModemManager
+    enabled: no
+    state: stopped
+
+- name: Create Docker config directory 
+  file:
+    path: /etc/docker 
+    state: directory
```

Comment:
need any specific permissions here?

Reply:
I was trying to match the install.sh script from the home-assistant repo, and they don't specify anything particular here.